nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-framework-cluster/src/test/java/org/apache/nifi/cluster/integration/Node.java

```diff
@@ -311,6 +320,11 @@ public boolean hasRole(final String roleName) {
         return leaderAddress.equals(getClusterAddress());
     }
 
+    public void addProcessor(final Processor example) throws ProcessorInstantiationException {
```

Comment:
Why is there a "addProcessor" method being added Node.java? There aren't any other "addX" methods and it seems weird to be "adding" it to a node.